UnrealPlugin/README.md

```diff
@@ -10,337 +11,38 @@ It was tested with Unreal Engine 4.26.2, but should work with other engines as w
-```
-
-If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
-
-If you are using Continuous Integration (CI), then you could add it to your setup to only turn this flag on when building a dedicated server, so the additional dlls only get added if it is a dedicated server build.
-
-## Packaging the game server
-
-Make sure that you have just built your project in development editor in Visual Studio *before* doing this next step, because sometimes building fails when opening the project directly through the .uproject file.
+## Requirements
 
-Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files". 
+* Download Visual Studio (the [community version is free](https://visualstudio.microsoft.com/vs/community/))
+	- Requires Workloads: .NET desktop development, and Desktop development with C++
+* Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal Engine website. This was tested on [4.26.2](https://github.com/EpicGames/UnrealEngine/releases/tag/4.26.2-release) (Link requires permissions)
+* Download the [Unreal PlayFabGSDK Plugin folder](https://github.com/PlayFab/gsdk/tree/master/UnrealPlugin)
+* [Optional] Download the [LocalMultiplayerAgent](https://github.com/PlayFab/MpsAgent/releases)
+	* [Optional] Alternately, download the [LocalMultiplayerAgent source](https://github.com/PlayFab/MpsAgent/tree/main/LocalMultiplayerAgent)
+* [Optional] [PlayFab Marketplace plugin](https://www.unrealengine.com/marketplace/product/playfab-sdk) or the [source version on GitHub](https://github.com/PlayFab/UnrealMarketplacePlugin/tree/master/4.26/PlayFabPlugin/PlayFab) (Not required for GSDK, but required for most other PlayFab services, including login)
 
-Then open the .uproject file.
+## Project creation
 
-In the top left of the Unreal editor menu, click on File -> Package -> Target Configuration -> [game name]Server and then go to File -> Package -> Windowsx64. 
-You can now use this packaged version of your game server to [test with LocalMultiplayerAgent](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab) 
-or to use it directly with PlayFab by [creating a build](https://developer.playfab.com/) and then using [PlayFab MpsAllocatorSample](https://github.com/PlayFab/MpsSamples/blob/master/MpsAllocatorSample/README.md).
+Unreal GSDK is installed into an Unreal server project. You will need a network-enabled multiplayer Unreal project with a dedicated-server mode. If you do not have one, you can follow our [Unreal prerequsitie setup](ThirdPersonMPSetup.md) guide. Once you have a network-enabled, multiplayer game, with a dedicated server, return here and continue.
```

Comment:
Open your Unreal server project, then install the Unreal GSDK. The Unreal server project needs to be a network-enabled multiplayer Unreal project with a dedicated-server mode. If you don't have one yet, follow our [Unreal prerequisite set up guide] (ThirdPersonMPSetup.md) to set one up. Once you have a network-enabled, multiplayer game, with a dedicated server, return to this step and continue.

Reply:
It's better to list the steps on how to open and install the GDSK. Saying it is good but telling them how to do it is better.

Reply:
I have applied two changes based on your suggestions here:
1. I replaced my initial block with yours. I then modified yours a bit, and reordered the sentences. Please take a look at the latest iteration.
2. I have tried to devise a checklist which describes each "word" of the requirement. In truth the requirement is fairly specific, but the implementation can vary ... infinitely. Two of our samples have wildly different implementations of the "network connected" requirement, but they're both valid. I've done my best to try and describe the requirements in terms of general capability, since a strict technical definition would be too limiting.